File: C++/1169.cpp
#include <iostream>

using namespace std;

int main(){
	int age;
	cin >> age;
	int bYear = 2012 - age + 1;
	if(bYear < 2000) cout << bYear - 1900 << " " << 1;
	else cout << bYear - 2000 << " " << 3;
}

File: Java/1169.java
import java.util.Scanner;

public class Main{
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int age = sc.nextInt();
        int bYear = 2012 - age + 1;
        
        if(bYear < 2000) System.out.println(bYear - 1900 + " " + 1);
        else System.out.println(bYear - 2000 + " " + 3);
    }
}